DEVELOPMENT.md

```diff
@@ -1,3 +1,35 @@
 # Developing additions to the Catalog
 
-TBD, for now see [CONTRIBUTING.md](CONTRIBUTING.md).
+The Catalog repository is intended to serve as a location where users can find
+`Task`s and `Pipeline`s that are maintained, useful and follow established
+best practices.
+
+When reviewing PRs that add new `Task`s or `Pipeline`s, maintainers will follow
+the following guidelines:
+
+* Submissions should be independently usable.
+While this repository is meant to be educational, its primary goal is to serve
+as a place users can find, share and discover useful components.
```

Comment:
I think an example here would be helpful.

Reply:
How's this?